### PR TITLE
Move Fava to beancount/fava

### PR DESCRIPTION
**Pull request by aumayr**
This PR tracks all changes neccessary before moving to https://github.com/beancount/fava (see #383)

Before moving, I want to collect the things that have to be changed:

- [X] https://aumayr.github.io/fava => https://beancount.github.io/fava
- [X] Update scripts for pythonanywhere.com
- [X] Help and docs
- [x] Move over the repo (https://help.github.com/articles/about-repository-transfers/)

Please comment if I missed something.

### CODE REVIEW

**Comment by aumayr** [comment]
Other questions that arose during making those changes:

- What about https://github.com/aumayr/beancount_urlscheme/ ?
- What about http://aumayr.github.io/beancount-sql-queries/ ?
- What about https://aumayr.github.io/beancount-docs-static/ ?

**Comment by yagebu** [comment]
Looks good!

> What about [...]

Don't have a strong opinion about moving those.

**Comment by aumayr** [comment]
@yagebu Move-date on Saturday (29th)?

**Comment by yagebu** [comment]
Sounds good!

File: README.rst [hunk 1]
```diff
@@ -1,21 +1,21 @@
-.. image:: https://badges.gitter.im/aumayr/fava.svg
-   :alt: Join the chat at https://gitter.im/aumayr/fava
-   :target: https://gitter.im/aumayr/fava
+.. image:: https://badges.gitter.im/beancount/fava.svg
+   :alt: Join the chat at https://gitter.im/beancount/fava
+   :target: https://gitter.im/beancount/fava
```
Comment (aumayr) [inline README.rst]: https://github.com/gitterHQ/gitter/issues/1488

**Comment by aumayr** [comment]
@yagebu I moved the repo a few minutes ago. Please reopen #383 if there are any issues.

**Comment by aumayr** [comment]
Should we make a new release as links in `setup.py` etc. have changed?

**Comment by yagebu** [comment]
> Should we make a new release as links in setup.py etc. have changed?

Not necessary I think. Regardless, it would be nice to setup a redirect from the old website (or at least have the link there to the new website)

**Comment by aumayr** [comment]
> Regardless, it would be nice to setup a redirect from the old website (or at least have the link there to the new website)

Added.

But I think it was stupid, as the repo transfer automatically created redirects on Github for all the issues, etc., but just not for the Github pages. Now I created a new repo under `aumayr/fava`, added HTML with a redirect for Github pages, and that works now, but all the issues redirects, etc. are gone.

**Comment by yagebu** [comment]
> But I think it was stupid, as the repo transfer automatically created redirects on Github for all the issues, etc., but just not for the Github pages. Now I created a new repo under aumayr/fava, added HTML with a redirect for Github pages, and that works now, but all the issues redirects, etc. are gone.

Yes, that's annoying. Stupid that Github doesn't just add a redirect for the pages too...

**Pull request merged**